proc_scripts/phasing.py

```diff
@@ -29,7 +29,8 @@ def zeroth_order_ph(d, fl=None):
     '''
     cov_mat = cov(c_[
         d.data.real.ravel(),
-        d.data.imag.ravel()].T,aweights=d.unit_weights)
```

Comment:
I don't expect that what you had would have worked

Reply:
(`unit_weights` is not defined)

Reply:
okay so I'm moving this to the issue with zeroth order phasing with a new pull request. So in this pull request this can be closed

Reply:
But I changed this in a subsequent commit, and it might work already

Reply:
And if you make a new pull request or issue like this, please reference here

Reply:
you gave thumbs up, but didn't reference the pull request you're talking about

Reply:
I haven't made a new pull request I am focused right now on getting this one closed.